rascal2/dialogs/settings_dialog.py

```diff
@@ -0,0 +1,116 @@
+from typing import Callable
+
+from PyQt6 import QtCore, QtWidgets
+
+from rascal2.core.settings import Settings, SettingsGroups
+from rascal2.widgets.inputs import ValidatedInputWidget
+
+
+class SettingsDialog(QtWidgets.QDialog):
+    def __init__(self, parent):
+        """
+        Dialog to adjust RasCAL-2 settings.
+
+        Parameters
+        ----------
+        parent : MainWindowView
+            The view of the RasCAL-2 GUI
+        """
+        super().__init__(parent)
+
+        self.setModal(True)
+        self.setMinimumWidth(400)
```

Comment:
I think you can go a bit bigger on the min width but also set a min height. something around w: 600, h: 400 looked good to me